playground/module-resolution/__tests__/module-resolution.spec.ts

```diff
@@ -3,11 +3,17 @@ import {
 	getJsonResponse,
 	getTextResponse,
 	isBuild,
+	page,
 	serverLogs,
+	viteTestUrl,
 } from '../../__test-utils__';
 
 describe.runIf(!isBuild)('module resolution', async () => {
-	afterAll(() => expect(serverLogs.errors).toEqual([]));
+	afterAll(() => {
+		const isNotUnconfiguredAliasError = (error: string) =>
+			!error.includes('@alias/unconfigured');
+		expect(serverLogs.errors.filter(isNotUnconfiguredAliasError)).toEqual([]);
```

Comment:
Is doing expectations in `afterAll()` an anti-pattern? How easy will it be to see which test caused the problems if the expectation fails?